@codex please fix

```
=================================== FAILURES ===================================
_____ test_main_subprocess_succeeds_for_packaged_layout_without_pythonpath _____

tmp_path = PosixPath('/tmp/pytest-of-runner/pytest-0/test_main_subprocess_succeeds_0')

    def test_main_subprocess_succeeds_for_packaged_layout_without_pythonpath(tmp_path):
        python_dir = tmp_path / 'bin' / 'resources' / 'python'
        import_root = tmp_path / 'bin' / 'resources' / '_up_' / '_up_'
        utils_dir = import_root / 'utils'
        python_dir.mkdir(parents=True)
        utils_dir.mkdir(parents=True)
    
        (python_dir / 'compute_node_bridge.py').write_text(
            MODULE_PATH.read_text(encoding='utf-8'),
            encoding='utf-8',
        )
        path_bootstrap_path = MODULE_PATH.parent / 'path_bootstrap.py'
        (python_dir / 'path_bootstrap.py').write_text(
            path_bootstrap_path.read_text(encoding='utf-8'),
            encoding='utf-8',
        )
        (utils_dir / '__init__.py').write_text('', encoding='utf-8')
        (utils_dir / 'compute_node_runtime.py').write_text(
            """
    SUPPORTED_COMPUTE_MODES = {"auto", "cpu", "cuda", "metal"}
    
    
    def normalize_compute_mode(mode):
        mode = str(mode).lower()
        return mode if mode in SUPPORTED_COMPUTE_MODES else "auto"
    
    
    def apply_compute_mode(_model_manager, mode):
        return normalize_compute_mode(mode)
    
    
    def resolve_relay_url(relay_url, **_kwargs):
        return relay_url
    
    
    def resolve_relay_port(relay_port, _relay_url):
        return relay_port
    
    
    def is_legacy_relay_payload(_payload):
        return False
    
    
    class ComputeNodeRuntimeConfig:
        def __init__(self, relay_url, relay_port, **_kwargs):
            self.relay_url = relay_url
            self.relay_port = relay_port
    
    
    class _RelayClient:
        def __init__(self, relay_url):
            self.relay_url = relay_url
    
    
    class _ModelManager:
        model_path = ""
    
    
    class ComputeNodeRuntime:
        def __init__(self, config):
            self.relay_client = _RelayClient(config.relay_url)
            self.model_manager = _ModelManager()
    
        def ensure_model_ready(self):
            return True
    
        def register_and_poll_once(self):
            return {"next_ping_in_x_seconds": 0}
    
        def process_relay_request(self, _payload):
            return True
    
        def stop(self):
            return None
    """.strip()
            + "\n",
            encoding='utf-8',
        )
    
        env = os.environ.copy()
        env.pop('PYTHONPATH', None)
        proc = subprocess.Popen(
            [
                sys.executable,
                str(python_dir / 'compute_node_bridge.py'),
                '--model',
                '/tmp/model.gguf',
                '--mode',
                'auto',
                '--relay-url',
                'https://token.place/',
            ],
            stdin=subprocess.PIPE,
            stdout=subprocess.PIPE,
            stderr=subprocess.PIPE,
            text=True,
            env=env,
        )
        assert proc.stdin is not None
        assert proc.stdout is not None
        assert proc.stderr is not None
        proc.stdin.write('{"type":"cancel"}\n')
        proc.stdin.flush()
        proc.stdin.close()
>       proc.wait(timeout=10)

tests/unit/test_desktop_compute_node_bridge.py:570: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.11.15/x64/lib/python3.11/subprocess.py:1264: in wait
    return self._wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <Popen: returncode: None args: ['/opt/hostedtoolcache/Python/3.11.15/x64/bin...>
timeout = 10

    def _wait(self, timeout):
        """Internal implementation of wait() on POSIX."""
        if self.returncode is not None:
            return self.returncode
    
        if timeout is not None:
            endtime = _time() + timeout
            # Enter a busy loop if we have a timeout.  This busy loop was
            # cribbed from Lib/threading.py in Thread.wait() at r71065.
            delay = 0.0005 # 500 us -> initial delay of 1 ms
            while True:
                if self._waitpid_lock.acquire(False):
                    try:
                        if self.returncode is not None:
                            break  # Another thread waited.
                        (pid, sts) = self._try_wait(os.WNOHANG)
                        assert pid == self.pid or pid == 0
                        if pid == self.pid:
                            self._handle_exitstatus(sts)
                            break
                    finally:
                        self._waitpid_lock.release()
                remaining = self._remaining_time(endtime)
                if remaining <= 0:
>                   raise TimeoutExpired(self.args, timeout)
E                   subprocess.TimeoutExpired: Command '['/opt/hostedtoolcache/Python/3.11.15/x64/bin/python', '/tmp/pytest-of-runner/pytest-0/test_main_subprocess_succeeds_0/bin/resources/python/compute_node_bridge.py', '--model', '/tmp/model.gguf', '--mode', 'auto', '--relay-url', 'https://token.place/']' timed out after 10 seconds

/opt/hostedtoolcache/Python/3.11.15/x64/lib/python3.11/subprocess.py:2045: TimeoutExpired
=============================== warnings summary ===============================
tests/unit/test_rate_limit.py::test_exceeding_api_rate_limit_returns_429
tests/unit/test_rate_limit.py::test_rate_limit_uses_openai_style_error_payload
tests/unit/test_rate_limit.py::test_streaming_chat_completion_requests_are_rate_limited
tests/unit/test_relay_configured_nodes.py::test_configured_nodes_endpoint_reflects_pool
tests/unit/test_relay_registration_token.py::test_sink_rejects_missing_registration_token
tests/unit/test_relay_registration_token.py::test_source_requires_registration_token
tests/unit/test_relay_registration_token.py::test_sink_accepts_plural_registration_tokens
  /opt/hostedtoolcache/Python/3.11.15/x64/lib/python3.11/site-packages/flask_limiter/extension.py:330: UserWarning: Using the in-memory storage for tracking rate limits as no storage was explicitly specified. This is not recommended for production use. See: [https://flask-limiter.readthedocs.io#configuring-a-storage-backend](https://flask-limiter.readthedocs.io/#configuring-a-storage-backend) for documentation about configuring the storage backend.
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```

new codex task, not a r/e/v/i/e/w task

--- a/tests/unit/test_desktop_compute_node_bridge.py
+++ b/tests/unit/test_desktop_compute_node_bridge.py
@@ -530,7 +530,7 @@ class ComputeNodeRuntime:
         return True
 
     def register_and_poll_once(self):
-        return {"next_ping_in_x_seconds": 0}
+        return {"next_ping_in_x_seconds": 1}
 
     def process_relay_request(self, _payload):
         return True

